content/docs/guides/usage-reporting.md

```diff
@@ -17,7 +17,18 @@ ks param set spartakus reportUsage false
 kubectl delete -n ${NAMESPACE} deploy spartakus-volunteer
 ```
 
-To explictly enable usage reporting repeat the above steps setting reportUsage to `true`
+**To disable usage reporting** you need to delete spartakus component. 
```

Comment:
Can you replace the text on line 11; with this line

"This is entirely voluntary and you can opt."

You can also delete the line

ks param set spartakus reportUsage false

since deleting the spartakus component is a more comprehensive change.